dashboard/src/components/DeploymentForm/DeploymentForm.test.tsx

```diff
@@ -95,5 +95,15 @@ it("renders the full DeploymentForm", () => {
   const wrapper = shallow(
     <DeploymentForm {...defaultProps} selected={{ versions, version: versions[0] }} />,
   );
+  // Force static name
+  wrapper.setState({ releaseName: "bar" });
   expect(wrapper).toMatchSnapshot();
 });
+
+it("renders a release name by default", () => {
```

Comment:
You could potentially mock Moniker to test that its generation is the value set and changes between mounts.

Reply:
I'd avoid all the `sinon`/`stub` integration just for that.

Reply:
Why is that? You do not need to use sinon but `jest`.

I think that the key here is if you care about testing that the value set as name changes and depends on an autogenerated value. Which is not being tested in your case.

Reply:
ah, well, in that case it's not necessary to mock the behavior just to check that the release name has a length bigger than 0 and that it changes.